apps/dotcms-ui/src/app/view/components/_common/forms/dot-push-publish-form/dot-push-publish-form.component.ts

```diff
@@ -109,6 +130,20 @@ export class DotPushPublishFormComponent
         );
     }
 
+    private loadTimezones(): void {
+        this.dotTimezoneService
+            .get()
+            .pipe(take(1))
+            .subscribe((timezones: DotTimeZone[]) => {
+                this.timeZoneOptions = timezones.map((item: DotTimeZone) => {
```

Comment:
Can this be an Observable and use async pipe?

Reply:
I don't think so, the data needs to be transformed and then that transformed data will be used as a reference to functions `updateTimezoneLabel` and `setUsersTimeZone`